packages/components/select/src/use-select.ts

```diff
@@ -792,6 +792,10 @@ export interface UseSelectOptionGroupProps extends HTMLUIProps<"ul"> {
    * The label of the option group.
    */
   label: string
+  /**
+   * Props for option group element.
+   */
+  labelProps?: HTMLUIProps<"span">
```

Comment:
Do not define props to customize components here.  Set to the `option-group.tsx` side.

Reply:
fixed